golangci.yml

```diff
@@ -90,7 +90,6 @@ linters:
     - paralleltest
     - thelper
```

Comment:
About revive, the default settings are very lame

There is an enable-all equivalent, that should be activated

Here is a golangci-lint .golangci.yml I wrote a few months ago

You can have a look at the revive settings

https://github.com/theredditbandit/pman/blob/master/.golangci.yaml#L133-L219

Reply:
You will have a lot of work if you enable it, but it will find so many errors you have no ideas they exist

Reply:
will fix in #56  this PR goal was to remove current warnings for 1.59.1 not to revamp the whole thing :)

Reply:
I pointed out you might have many errors you would like to address. So maybe it would be be better to start fixing them locally first.

But I get your point